README.md

```diff
@@ -1 +1,3 @@
-# sahar-exercise
\ No newline at end of file
+# sahar-exercise
```

Comment:
// TODO: move this repo to you personal account later by forking. Also might be a good idea to install linters and lint configs, and put the unit tests for the exercise in a folder and create a npm script in the package.json to be able run the unit.